app/src/main/res/values-night/colors.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+    <color name="primary">@color/grey300</color>
+    <color name="primaryVariant">@color/black</color>
```

Comment:
use default android color for black `@android:color/black`. Do this to other files